dropbox/oauth.py

```diff
@@ -203,7 +238,7 @@ class DropboxOAuth2FlowNoRedirect(DropboxOAuth2FlowBase):
         dbx = Dropbox(oauth_result.access_token)
     """
 
-    def __init__(self, consumer_key, consumer_secret, locale=None):  # noqa: E501; pylint: disable=useless-super-delegation
+    def __init__(self, consumer_key, consumer_secret, locale=None, token_access_type='legacy'):  # noqa: E501; pylint: disable=useless-super-delegation
```

Comment:
Worth adding examples in doc string about how to use short-live token and label that as alpha feature that subject to change. And if they want to use this feature. Contact dropbox support